PyGMT test breakages on Ghostscript 9.54.0

**Description of the problem**

<!-- Please be as detailed as you can when describing an issue. The more information we have, the easier it will be for us to track this down. -->

[Ghostscript 9.54.0](https://www.ghostscript.com/doc/9.54.0/News.htm) released on conda-forge 11 May 2021 (see https://anaconda.org/conda-forge/ghostscript/files) seems to have broken some unit tests due to tiny image differences. We'll need to update the baseline images on DAGsHub.

| baseline (old) | result (new) | diff |
|--|--|--|
| ![baseline](https://user-images.githubusercontent.com/23487320/117903462-b82b3a00-b323-11eb-87b8-f07c7846575a.png) | ![result](https://user-images.githubusercontent.com/23487320/117903467-b95c6700-b323-11eb-8f75-75fba949c77e.png) | ![result-failed-diff](https://user-images.githubusercontent.com/23487320/117903473-b9f4fd80-b323-11eb-9622-dc3c97cf8645.png) |


**Full error message**

https://github.com/GenericMappingTools/pygmt/runs/2561524507?check_suite_focus=true#step:11:471

<details>

```python-traceback
=================================== FAILURES ===================================
__________________________ test_basemap_winkel_tripel __________________________
Error: Image files did not match.
  RMS Value: 3.641081499776249
  Expected:  
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_basemap.test_basemap_winkel_tripel/baseline.png
  Actual:    
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_basemap.test_basemap_winkel_tripel/result.png
  Difference:
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_basemap.test_basemap_winkel_tripel/result-failed-diff.png
  Tolerance: 
    2
_______________________ test_config_map_grid_cross_size ________________________
Error: Image files did not match.
  RMS Value: 11.811434080255305
  Expected:  
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_grid_cross_size/baseline.png
  Actual:    
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_grid_cross_size/result.png
  Difference:
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_grid_cross_size/result-failed-diff.png
  Tolerance: 
    2
___________________________ test_config_map_grid_pen ___________________________
Error: Image files did not match.
  RMS Value: 14.442821202930824
  Expected:  
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_grid_pen/baseline.png
  Actual:    
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_grid_pen/result.png
  Difference:
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_grid_pen/result-failed-diff.png
  Tolerance: 
    2
_________________________ test_config_map_tick_length __________________________
Error: Image files did not match.
  RMS Value: 14.999485446980628
  Expected:  
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_tick_length/baseline.png
  Actual:    
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_tick_length/result.png
  Difference:
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_tick_length/result-failed-diff.png
  Tolerance: 
    2
___________________________ test_config_map_tick_pen ___________________________
Error: Image files did not match.
  RMS Value: 15.210231972644442
  Expected:  
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_tick_pen/baseline.png
  Actual:    
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_tick_pen/result.png
  Difference:
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_config.test_config_map_tick_pen/result-failed-diff.png
  Tolerance: 
    2
```
</details>

**System information**

Please paste the output of `python -c "import pygmt; pygmt.show_versions()"`:

```
PyGMT information:
  version: v0.3.2.dev135+g036e2b0
System information:
  python: 3.9.4 | packaged by conda-forge | (default, May 10 2021, 22:13:15)  [Clang 11.1.0 ]
  executable: /Users/runner/miniconda3/envs/pygmt/bin/python
  machine: macOS-10.15.7-x86_64-i386-64bit
Dependency information:
  numpy: 1.20.2
  pandas: 1.2.4
  xarray: 0.18.0
  netCDF4: 1.5.6
  packaging: 20.9
  ghostscript: 9.54.0
  gmt: 6.2.0rc1
GMT library information:
  binary dir: /Users/runner/miniconda3/envs/pygmt/bin
  cores: 3
  grid layout: rows
  library path: /Users/runner/miniconda3/envs/pygmt/lib/libgmt.dylib
  padding: 2
  plugin dir: /Users/runner/miniconda3/envs/pygmt/lib/gmt/plugins
  share dir: /Users/runner/miniconda3/envs/pygmt/share/gmt
  version: 6.2.0rc1
```

--- a/pygmt/tests/baseline/test_basemap_winkel_tripel.png.dvc
+++ b/pygmt/tests/baseline/test_basemap_winkel_tripel.png.dvc
@@ -1,4 +1,4 @@
 outs:
-- md5: b32f987475dee83c219df916a7ffb687
-  size: 62786
+- md5: 128e58b420ec614da56fd273c4f4c1ea
+  size: 62767
   path: test_basemap_winkel_tripel.png

--- a/pygmt/tests/baseline/test_config_map_grid_cross_size.png.dvc
+++ b/pygmt/tests/baseline/test_config_map_grid_cross_size.png.dvc
@@ -1,4 +1,4 @@
 outs:
-- md5: cd5aec2937c08cf7d94f80eb6894609a
-  size: 13246
+- md5: c154f4630cd8ec5ceaf5329fdcc6aa5c
+  size: 13550
   path: test_config_map_grid_cross_size.png

--- a/pygmt/tests/baseline/test_config_map_grid_pen.png.dvc
+++ b/pygmt/tests/baseline/test_config_map_grid_pen.png.dvc
@@ -1,4 +1,4 @@
 outs:
-- md5: eb591a3581ad0cc82ff08fd4aa763b8c
-  size: 13506
+- md5: 3a3423437fcc7039db04e4c1bbb779e7
+  size: 13562
   path: test_config_map_grid_pen.png

--- a/pygmt/tests/baseline/test_config_map_tick_length.png.dvc
+++ b/pygmt/tests/baseline/test_config_map_tick_length.png.dvc
@@ -1,4 +1,4 @@
 outs:
-- md5: 2fdbc52afba87ecf8c434e4b62232455
-  size: 12664
+- md5: b20b1fd9f9e585abb5da08f3201e69ce
+  size: 12965
   path: test_config_map_tick_length.png

--- a/pygmt/tests/baseline/test_config_map_tick_pen.png.dvc
+++ b/pygmt/tests/baseline/test_config_map_tick_pen.png.dvc
@@ -1,4 +1,4 @@
 outs:
-- md5: 380b4c6fb0245a393aeb76b5c1f6ee91
-  size: 13085
+- md5: ebc85cc6c78a30b1d55972a725b01005
+  size: 13502
   path: test_config_map_tick_pen.png

